app/src/main/java/com/mapbox/vision/examples/activity/main/MainActivity.kt

```diff
@@ -181,6 +182,16 @@ class MainActivity : AppCompatActivity() {
         override fun estimatedPositionUpdated(position: Position) {}
 
         override fun calibrationProgressUpdated(calibrationProgress: CalibrationProgress) {}
+
+        override fun laneDepartureStateUpdated(laneDepartureState: LaneDepartureState) {
+            if (currentMode == LINE_DETECTION_MODE) {
```

Comment:
As for now sounds will be played without any UI indication? IMO its kinda very confusing...